dist/js/gumshoe.js

```diff
@@ -24,13 +24,15 @@
 	//
 
 	var gumshoe = {}; // Object for public APIs
-	var supports = !!document.querySelector && !!root.addEventListener; // Feature test
+	var supports = !!document.querySelectorAll && !!root.addEventListener; // Feature test
```

Comment:
I updated this to check for `querySelectorAll` as your main `navLinks` selector requires it so it felt better to update this check to match.

```js
var navLinks = document.querySelectorAll( settings.selector );
```